deltachat-ios/AppDelegate.swift

```diff
@@ -24,13 +17,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
     private var backgroundTask: UIBackgroundTaskIdentifier = .invalid
     var reachability = Reachability()!
     var window: UIWindow?
-    var state = ApplicationState.stopped
     var appIsInForeground = false
 
     func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // explicitly ignore SIGPIPE to avoid crashes, see https://developer.apple.com/library/archive/documentation/NetworkingInternetWeb/Conceptual/NetworkingOverview/CommonPitfalls/CommonPitfalls.html
+        // setupCrashReporting() may create an additional handler, but we do not want to rely on that
```

Comment:
indeed, DBDebugToolkit is exactly doing that 
```
    // SIGPIPE signal
    _previousSIGPIPEHandler = signal(SIGPIPE, SIG_IGN);
    signal(SIGPIPE, handleSIGPIPESignal);
```

Reply:
yi., however, if we disable/move the setupCrashReporting or maybe DBDebugToolkit gets an update and does things differently, i think it is good to have this disabled explicitly.